packages/wasm-ast-types/src/react-query.spec.ts

```diff
@@ -57,5 +57,12 @@ it('createReactQueryHooks', () => {
       'Sg721',
       'Sg721QueryClient'
     )))
+  expectCode(t.program(
+    createReactQueryHooks(
+      query_msg,
+      'Sg721',
+      'Sg721QueryClient',
+      { optionalClient: true }
+    )))
```

Comment:
I think `createReactQueryHooks` takes an object?

Maybe we can replace this test case with:

```js
describe('createReactQueryHooks', () => {
  it('no options', () => {
    expectCode(t.program(
      createReactQueryHooks({
        queryMsg: query_msg,
        contractName: 'Sg721',
        QueryClient: 'Sg721QueryClient'
      })));
  });
  it('optionalClient: false', () => {
    expectCode(t.program(
      createReactQueryHooks({
        queryMsg: query_msg,
        contractName: 'Sg721',
        QueryClient: 'Sg721QueryClient',
        options: { optionalClient: false }
      })));
  })
});

```

Reply:
It's optional so it works without the parameter but I can make it required.

Reply:
oh — we can make `options` optional — but just give it defaults like I was mentioning before.

Reply:
like this

```js
export const createReactQueryHooks = ({
    queryMsg,
    contractName,
    QueryClient,
    options = { optionalClient: false }
}: ReactQueryHooks) => {
```

Reply:
Sure. Now it's an object. Funny enough I did and then undid this [here](https://github.com/CosmWasm/ts-codegen/pull/31/commits/5c5d9b1a3e0a7eca2c9af43938090723442b9c35#diff-4b3303ef14bc8d8253c59270a7611661ef35dbbb1b3df79b1360dd7160610901)